integrated_channels/integrated_channel/transmitters/learner_data.py

```diff
@@ -80,10 +80,28 @@ def single_learner_assessment_grade_transmit(self, exporter, **kwargs):
                 enterprise_customer_uuid,
                 lms_user_id,
                 learner_data.course_id,
-                f'create_assessment_reporting started for enrollment {learner_data.enterprise_course_enrollment_id}'
+                'create_assessment_reporting started for '
+                f'integrated_channel_enterprise_enrollment_id={learner_data.enterprise_course_enrollment_id}'
```

Comment:
more use of splunk variable formatting. this will make log parsing of dry-runs WAY easier